src/components/expanding-panel/ExpandingPanel.scss

```diff
@@ -6,7 +6,7 @@
   align-items: center;
 
   button {
-    background-color: var(--secondary-blue);
+    /*background-color: var(--secondary-color);*/
```

Comment:
Can we remove this commented out style here?

Reply:
yes will remove that